code/go/0chain.net/chaincore/chain/entity.go

```diff
@@ -570,6 +570,12 @@ func (c *Chain) setupInitialState(initStates *state.InitStates) util.MerklePatri
 	state := cstate.NewStateContext(nil, pmt, nil, nil, nil, nil, nil, nil, nil)
 	mustInitPartitions(state)
 
+	_, err := minersc.GetGlobalNode(state)
```

Comment:
`minersc.GetGlobalNode` it self would not store the config to the state, though it also should not save state to it as `get` action should not change anything. What we need to do in this PR is something like:

```go
if err := minersc.InitConfig(state); err != nil {
    panic(err)
}
if err := storagesc.InitConfig(state); err != nil {
    panic(err)
}

...
and other smart contract packages.
```